metecho/api/models.py

```diff
@@ -557,6 +572,7 @@ class Task(
     branch_name = models.CharField(
         max_length=100, null=True, blank=True, validators=[validate_unicode_branch]
     )
+    org_config_name = StringField()
```

Comment:
@wlonk Since the valid options here are hardcoded in `jobs.py` ('dev' | 'feature' | 'qa' | 'beta' | 'release'), would it make sense to validate this value, probably in the serializer?

Reply:
Those are just the valid fallback options; other options are valid if the project config is appropriately formed.